calculator.py

```diff
@@ -70,4 +90,4 @@ def main():
 
 # call main function
 if __name__ == "__main__":
-    main()
+    main()
```

Comment:
The code stops after every succesful execution which is alittle less effective.

You can ask if the user wants to continue or do another calculation, if yes...the program re-runs, if no, the program stops.

here is a basic approach to it ,
def main():
    cont = True
    while cont:
        first_number = get_number("first number")
        if first_number is None:
            return

        second_number = get_number("second number")
        if second_number is None:
            return

        operation, symbol = get_operation()

        result = execute(operation, first_number, second_number)
        print(f"Result: {first_number} {symbol} {second_number} = {result}")
        cont_input = input("Would you like to perform another operation? yes or y to continue and no or n to stop :")
        if cont_input == "yes" or cont_input == "y":
            continue
        else:
            print("Have a great day")
            cont = False

Reply:
Noted. Thanks boss